_posts/uportal/2017-07-05-uportal-slack-summary.md

```diff
@@ -0,0 +1,73 @@
+---
+layout:     post
+title:      June 2017 uPortal Slack summary
+summary:    Summarizing Slack traffic about uPortal in June 2017.
+tags:       [uPortal]
+---
+
+In June 2017, 11 people participated substantially in conversation in the [slack.apereo.org #uportal channel].
+
++ Aaron Grant
++ Andrew Petro
++ Brandon Powell
++ Christian Cousquer
++ Christian Murphy
++ David Sibley
++ Drew Wills
++ Jim Helwig
++ Marissa Warner-Wu
++ Matt Clare
++ Tim Vertein
+
+
+Conversation highlights include:
+
++ Aaron Grant's [uportal_commit_stats] script and its findings about uPortal contributions. [gitinspector] might also be useful for uPortal-related Git history analysis.
++ Internationalization of documentation stored in Git. What Symfony does in its [primary][symfony-docs] and [French-translation][symfony-docs-fr] documentation. (Separate repos.). Alternatively, the [jekyll-multiple-languages-plugin] . Example [ReactJS portlet with i18n support] based in [i18next].
++ Sharing resources from the Open Apereo 2017 conference, including [Portal?! uPortal! What is this Béchamel], [Speeding up uPortal with ReactJS], [two-hard-problems]. (Cf. [uportal-dev@ thread re conference artifacts].)
++ Encouraging interest in the new [Accessibility group]. (Cf. [Accessibility Across Apereo thread on uportal-user@])
++ [styled-components] for styling React.js apps, [browser support for Polymer].
++ [conversational UI example]
++ Weather API options, including World Weather Online, Weather Underground, weather.gov, DarkSky API.
++ The [@uPortal] Twitter account.
++ The role of some CSS classes ` <xsl:attribute name="class">up-portlet-control focus externalLink</xsl:attribute>` (turns out to be used in some JavaScript associated with the maximize menu option).
++ uPortal 5 [fanciness for setting up Tomcat][uPortal-start PR 6] and [deployment via Gradle][uPortal-start PR 8]. [Renaming the overlays directory][uPortal-start PR 10] (cf. [uportal-dev@ thread on overlays directory renaming] or [another][another uportal-dev@ thread on overlays directory renaming]).
++ [myday cloud-based portal]
++ [sonar][sonarwhal], a linting tool for the web
+
+
+
+## On Slack
+
+I have some [concerns about the openness properties of Slack as implemented by Apereo][open@ 2017-06-15]. You can't read its logs anonymously; you have to log in to access it and you can't come across its content via a Google search. Only the most recent ten thousand messages are available, so in practice this means logs are lost about a month back.
+
+Summarizing the conversations in this anonymously, publicly accessible and Google indexable context somewhat mitigates these problems.
```

Comment:
Gitter seems like it could be a good alternative to address the afore concerns.
ref: https://en.wikipedia.org/wiki/Gitter
ref: https://twitter.com/gitchat/status/596573614415204352